sdk/communication/Azure.Communication.Common/src/PhoneNumberIdentifier.cs

```diff
@@ -9,21 +9,26 @@ namespace Azure.Communication
     /// <summary>Represents a Phone Number.</summary>
     public class PhoneNumberIdentifier : CommunicationIdentifier
     {
+        /// <summary>The id of the phone number.</summary>
+        public string? Id { get; }
+
         /// <summary>The phone number in E.164 format.</summary>
         public string PhoneNumber { get; }
 
         /// <summary> Initializes a new instance of <see cref="PhoneNumberIdentifier"/>.</summary>
         /// <param name="phoneNumber">The phone number in E.164 format.</param>
+        /// <param name="id">Id of the Phone Number.</param>
```

Comment:
nit: casing and let's use "full"
```suggestion
        /// <param name="id">Full id of the phone number.</param>
```